docs/source/smooth_quant.md

```diff
@@ -28,7 +28,7 @@ where $X_{fp32}$ is the input matrix, $S$ is the scale factor,  $Z$ is the integ
 
 There are several choices of sharing quantization parameters among tensor elements, also called quantization granularity. The coarsest level, per-tensor granularity, is that all elements in the tensor share the same quantization parameters. Finer granularity means sharing quantization parameters per row or per column for 2D matrices and per channel for 3D matrices. Similarly, the finest granularity is that each element has an individual parameter.
```

Comment:
The last sentence, shall it be more suitable to say "... each element has a individual **set** of quantization parameters", since quantization parameters consist of scale and zero point, etc.